crowd-server/crowd_server/apps/food_fact/serializers.py

```diff
@@ -0,0 +1,37 @@
+from rest_framework import serializers
+from .models import *
+
+
+class QuestionSerialiser(serializers.ModelSerializer):
+
+
+    class Meta:
+        
+        model = Question
+        fields = '__all__'          # ->['language','img_url' ,'question','response_count']
+
+   
+
+
+class ResponseSerializer(serializers.ModelSerializer):
+    
+    class Meta:
+        model = Response
+        fields = '__all__'          # ->[ user, response, question ]
+
+        extra_kwargs={
+        'user' : {'require':True},
+        'response' : {'require':True},
+        'question' : {'require':True},
+        }
+        
+        
+    def create(self, validated_data):
+
+        question_content = validated_data['question']
+        question = Question.objects.get(id = question_content.id)
```

Comment:
The question_content has an instance of the Question class
Please consider the below example:
`def create(self, validated_data):`
  `      image = validated_data['image']`
`        image.answer_count += 1` 
`       image.save()` 
`        return ImageLabel.objects.create(**validated_data)`

Reply:
thank you for review :)